test/mock/outline.mock.js

```diff
@@ -0,0 +1,16 @@
+define([], function () {

```

Comment:
It's not at all important, but an interesting note: you can drop the `[]` entirely if you have no dependencies (as here, and a few other places).